### ISSUE TITLE
Deploying the App

### ISSUE THREAD

**Issue by iharshit009** [iharshit009/SecretPeek#6]
Usually, a flask or web app is deployed to Heroku, AWS, and other such cloud services. Here we need to take care that our server is not an HTTP/HTTPS web server. Here WebSocket's Server comes into play.

**Comment by iharshit009** [comment]
@Rits1272 would like to take this issue?

**Comment by Rits1272** [comment]
Yeah sure @iharshit009. I will make a PR for this after completing #5 :)

**Comment by iharshit009** [comment]
Okay, Go on

**Comment by codeit13** [comment]
I can deploy this web app on Heroku.
I will create a pipeline from GitHub to Heroku which will automatically push the changes from GitHub to Heroku whenever any push is made to Github.

**Comment by iharshit009** [comment]
Okay you could go ahead with it. Let me know what would you need

**Comment by codeit13** [comment]
Okay. Can I get Heroku credentials where I can deploy the web app?

**Comment by iharshit009** [comment]
I Will share with you soon, till then you could work on modifying the project structure

**Comment by codeit13** [comment]
Can I proceed with this? @iharshit009 
I have deployed my forked repo here https://secretpeek.herokuapp.com/
